veggieseasons/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -386,9 +386,10 @@
 					"$(inherited)",
 					"$(PROJECT_DIR)/Flutter",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = dev.flutter.VeggieSeasons;
```

Comment:
This is a published app, so we can't change the Bundle ID.

Reply:
Reverted Bundle ID change